sdk/storage/azure-storage-blob/azure/storage/blob/_container_client.py

```diff
@@ -1295,7 +1295,7 @@ def download_blob(
             **kwargs)
 
     @distributed_trace
-    def delete_blobs(
+    def delete_blobs(  # pylint: disable=delete-operation-wrong-return-type
```

Comment:
`azure/storage/blob/_container_client.py:1298: [C4752(delete-operation-wrong-return-type), ContainerClient.delete_blobs] delete* or begin_delete* should return None or LROPoller[None], respectively.`